### PR TITLE
Update pr_check to use more common code

### PR DESCRIPTION
**Pull request by bsquizz**

### CODE REVIEW

File: pr_check.sh [hunk 1]
```diff
@@ -1,107 +1,21 @@
...
 
-if [ ! -d bonfire ]; then
-    git clone https://github.com/RedHatInsights/bonfire.git
-fi
+source $APP_ROOT/unit_test.sh
 
-if [ ! -d venv ]; then
-    python3 -m venv venv
-fi
+#source $CICD_ROOT/deploy_ephemeral_env.sh
```
Comment (coderbydesign) [inline pr_check.sh:19]: Can these commented lines be removed?

Comment (bsquizz) [reply pr_check.sh:19]: We could. I'm using it as a place-holder related to running smoke tests to show at which point in the pr_check.sh it should be placed, we'll be un-commenting it soon

Comment (bsquizz) [reply pr_check.sh]: Removed them

**Review by coderbydesign — APPROVED** [review]

**Comment by bsquizz** [comment]
Rebased

**Review by catastrophe-brandon — APPROVED** [review]

**Pull request merged**